roles/qe_common/README.md

```diff
@@ -0,0 +1,58 @@
+qe_common
+=========
+
+The tests in this role are not specific to any one functional area.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+This role should not have any default variables as this is a common role for tests that can be used for multiple test roles.
+
+Dependencies
+------------
+
+None
```

Comment:
These assume that there is some openshift environment available.

The openstack, oc, kubectl, podman and virsh CLI tools are expected to be available.